components/support/utils/src/test/java/mozilla/components/support/utils/DomainMatcherTest.kt

```diff
@@ -61,6 +67,11 @@ class DomainMatcherTest {
             DomainMatch("http://firefox.com", "firefox.com"),
             segmentAwareDomainMatch("fire", urls)
         )
+//        // Path segments
+//        assertEquals(
+//            DomainMatch("bcferries.com/current-conditions", "bcferries.com/current-conditions"),
+//            segmentAwareDomainMatch("bcferries.com/cu", urls)
+//        )
```

Comment:
Just to confirm that this change was intentional ? :)

Reply:
This one was very unintentional 😂 Thanks for calling it out!